extensions/wikia/HTTPSSupport/HTTPSSupportHooks.class.php

```diff
@@ -70,8 +70,18 @@ public static function parserUpgradeSpecialFilePathURLs( string &$url ) {
 	 * @return boolean
 	 */
 	public static function onLinkerMakeExternalLink( string &$url, string &$text, bool &$link, array &$attribs ): bool {
-		if ( wfHttpsAllowedForURL( $url ) ) {
-			$url = wfProtocolUrlToRelative( $url );
+		global $wgFandomBaseDomain, $wgWikiaOrgBaseDomain, $wgWikiaBaseDomain;
+
+		$host = parse_url( $url, PHP_URL_HOST );
+
+		$normalizedHost = wfNormalizeHost( $host );
+		$explodedHost = explode( '.', $normalizedHost );
+
+		$pattern = '/('. $wgFandomBaseDomain .'|'. $wgWikiaOrgBaseDomain .'|(?:wikia|fandom)-dev.(?:com|pl|us))/';
+
+		if ( preg_match( $pattern, $host ) || ( preg_match( '/' . $wgWikiaBaseDomain . '/', $normalizedHost ) && sizeof( $explodedHost ) <= 3 ) )
```

Comment:
if you use `$normalizedHost` instead of `$host` in the first `preg_match` you don't have to care about dev domain in the regex